src/Schema/AbstractSchemaManager.php

```diff
@@ -694,7 +694,10 @@ public function alterSchema(SchemaDiff $schemaDiff): void
      */
     public function migrateSchema(Schema $toSchema): void
     {
-        $this->alterSchema(Comparator::compareSchemas($this->createSchema(), $toSchema));
+        // @phpstan-ignore-next-line
```

Comment:
We usually suppress errors in the configuration `phpstan.xml.dist`, please move this one there.